content/posts/2023/reana-0.9.2.md

```diff
@@ -0,0 +1,87 @@
+In particular, the usage of the database has been optimised in order to reduce the time needed to perform some of the most common operations, including listing and filtering workflows and jobs.
+
+These improvements can be quite noticeable, even more so when users have executed tens of thousands of workflows and jobs.
+For example, we have measured a performance improvement of around 18x (18 times, not percent!) when listing all the details of a thousand workflows with `reana-client-go`.
+
+### Launch-on-REANA badge generator
+
+REANA 0.9.0 has introduced the possibility to construct custom URLs which can be used to easily launch on REANA analyses that are hosted on remote publicly-accessible sources such as GitHub, GitLab or Zenodo.
+These URLs can then be shared with other users or used to create clickable badges that can be embedded, for example, in the README of your Git repository.
+
+With the new 0.9.2 release, REANA now provides a convenient way to construct these URLs and badges directly from the web interface. You just need to provide the URL to your analysis, plus some optional details like the desired name of the workflow, the path to the REANA specification file, or custom parameters. REANA will then show you the constructed URL and the code snippet that you can use to embed the Launch-on-REANA badge in your Markdown files.
+
+### ... and more!
+
+There are more small improvements in this release.
+It's now possible to delete all the runs of a given workflow directly from the web interface. Furthermore, the validation of the REANA specification file has been improved in case of invalid YAML files or unexpected keywords.
+
+Please see the detailed [REANA 0.9.2 release notes](https://github.com/reanahub/reana/releases/tag/0.9.2) for the complete list of all user-oriented changes.
+
+## What's new for the administrators?
+
+### Automounting of CVMFS repositories
+
+While users have been able to mount and access CVMFS repositories from their jobs, it was not possible to configure and access additional repositories other than the ones allowed by REANA.
+
+Thanks to CVMFS CSI v2, REANA 0.9.2 allows users to mount any available repository, as long as CVMFS is correctly configured in the Kubernetes cluster. To learn more about this, you can check out the official docs of CVMFS CSI on [adding CVMFS repository configuration](https://github.com/cvmfs-contrib/cvmfs-csi/blob/master/docs/how-to-use.md#adding-cvmfs-repository-configuration).
+
+### ARM Docker images
+
+Given the rise in interest for ARM computers, both for personal use and in servers, REANA now supports being deployed on ARM machines.
+Starting from this release, the REANA's Docker images published on [DockerHub](https://hub.docker.com/u/reanahub) will contain two variants, supporting `linux/amd64` and `linux/arm64` platforms. This change is transparent, and the correct variant will be automatically chosen by Docker depending on the machine it's running on.
```

Comment:
Starting from this release, we are publishing multi-platform container infrastructure images on DockerHub with two supported variants, linux/adm64 and linux/arm64. This change is fully transparent and the correct image variant will be automatically chosen when pulling the image based on the machine architecture where REANA is being deployed.